README.md

```diff
@@ -89,7 +89,7 @@ a parameter type.
 | ------------- | ----------- |
 | `name`        | The name the parameter type will be recognised by in output parameters.
 | `regexp`      | A regexp that will match the parameter. May include capture groups.
-| `type`        | The return type of the transformer {{% stepdef-body %}}.
+| `type` / `param_type` (Python)        | The return type of the transformer {{% stepdef-body %}}.
```

Comment:
Please remove `{{% stepdef-body %}}` - that's a leftover from an abaondoned documentation tool.

Reply:
Why did you choose to name it `param_type`? I don't think `type` is a reseved keyword in Python.

Reply:
Will address point 1.

RE point 2, I don't think it's "wrong" per se, but I thought it was bad practice to use type as a func param.  Happy to stand corrected if any pythonistas correct me, though!

See: https://stackoverflow.com/questions/10568087/is-it-safe-to-use-the-python-word-type-in-my-code  

Happy to review this accordingly